custom-steps/libpng/post-build.ps1

```diff
@@ -0,0 +1,19 @@
+param (
+    [Parameter(Mandatory=$true)][string]$BuildArtifactsPath
+)
+
+Import-Module "$PSScriptRoot/../../ps-modules/Util"
+
+$IsOnMacOS = Get-IsOnMacOS
+
+if(-not $IsOnMacOS) {
+    Write-Host "The step is only required for MacOS.  Skipping step..."
+    exit
+}
+
+Import-Module "$PSScriptRoot/../../ps-modules/MacUtil"
+
+Write-Host "Running post-build script..."
```

Comment:
I think we can get rid of this too. We don't need every post-build script saying that it's running. If we really want to, we can make the calling code say "running X post build script" but I think we should try to make each package-specific script as succinct as possible

Reply:
Fixed: e5d93971c0d8e3fb07bea1f99163f42b6d52d2e1